packages/venia-ui/lib/components/Image/image.js

```diff
@@ -1,18 +1,26 @@
 import React, { Fragment, useMemo } from 'react';
-import { func, shape, string } from 'prop-types';
+import PropTypes, { func, number, oneOfType, shape, string } from 'prop-types';
 import { useImage } from '@magento/peregrine/lib/talons/Image/useImage';
+import { transparentPlaceholder } from '@magento/peregrine/lib/util/images';
+import { resourceUrl } from '@magento/venia-drivers';
 
 import { generateSrcset } from '../../util/images';
 import { mergeClasses } from '../../classify';
 import defaultClasses from './image.css';
 
 /**
  * The Image component renders a placeholder until the image is loaded.
- * @param {string} prop.alt the alt text for the image
- * @param {string} props.classes any classes to apply to this component
- * @param {string} prop.placeholder the placeholder source to display while the image loads or if it errors on load
- * @param {string} props.src the source of the image
- * @param {string} props.fileSrc the raw source of the image without width and height added
+ *
+ * @param {string}   props.alt the alt text for the image
+ * @param {string}   props.classes any classes to apply to this component
+ * @param {number}   props.height the desired height of the image
+ * @param {function} props.onError callback for error loading image
+ * @param {function} props.onLoad callback for when image loads successfully
+ * @param {string}   props.placeholder the placeholder source to display while the image loads or if it errors on load
+ * @param {string}   props.resource the Magento path to the image ex: /v/d/vd12-rn_main_2.jpg
```

Comment:
If you keep the props, add `resourceWidth` and `resourceHeight` to these docs.

Reply:
Yes, thank you: https://github.com/magento/pwa-studio/pull/1871/commits/50761a370eaf396a6eec1e489e3766b2f430e096.